frontend/lib/verification/verification_hasher.dart

```diff
@@ -1,33 +0,0 @@
-import 'dart:convert';
```

Comment:
This file was moved to `identification/verification_workflow/verification_hasher.dart`